spec.html

```diff
@@ -47640,30 +47641,10 @@ <h1>Changes to IsLooselyEqual</h1>
 
       <emu-annex id="sec-IsHTMLDDA-internal-slot-typeof">
         <h1>Changes to the `typeof` Operator</h1>
-        <p>The following table entry is inserted into <emu-xref href="#table-typeof-operator-results"></emu-xref> immediately preceding the entry for "Object (implements [[Call]])":</p>
-        <emu-table>
-          <emu-caption>
-            Additional <emu-xref href="#sec-typeof-operator">`typeof`</emu-xref> Operator Results
-          </emu-caption>
-          <table>
-            <tr>
-              <th>
-                Type of _val_
-              </th>
-              <th>
-                Result
-              </th>
-            </tr>
-            <tr>
-              <td>
-                Object (has an [[IsHTMLDDA]] internal slot)
-              </td>
-              <td>
-                *"undefined"*
-              </td>
-            </tr>
-          </table>
-        </emu-table>
+        <p>The following steps replace step <emu-xref href="#step-typeof-web-compat-insertion-point"></emu-xref> of <emu-xref href="#sec-typeof-operator-runtime-semantics-evaluation">the evaluation semantics for `typeof`</emu-xref>:</p>
```

Comment:
```suggestion
        <p>The following step replaces step <emu-xref href="#step-typeof-web-compat-insertion-point"></emu-xref> of <emu-xref href="#sec-typeof-operator-runtime-semantics-evaluation">the evaluation semantics for `typeof`</emu-xref>:</p>
```

Reply:
We use the plural form when referring to single-step algorithms all the time: https://tc39.es/ecma262/#sec-date.prototype.valueof

I prefer the consistency of phrasing. It's 1 or more steps.

Reply:
> I prefer the consistency of phrasing. It's 1 or more steps.

I think in that case it sounds fine because of the "or more". In this case though, I prefer @bakkot's suggestion.